internal/daemon/daemon.go

```diff
@@ -433,6 +437,17 @@ func (d *Daemon) StartAPI(bootstrap bool, initConfig map[string]string, newConfi
 			return err
 		}
 
+		// Start all extension servers before post-bootstrap hook.
+		for _, extensionServer := range d.extensionServers {
+			server := d.initServer(extensionServer.Resources...)
+			url := api.NewURL().Scheme(extensionServer.Protocol).Host(extensionServer.Address.String())
+			network := endpoints.NewNetwork(d.shutdownCtx, endpoints.EndpointNetwork, server, *url, extensionServer.Certificate)
+			err = d.endpoints.Add(network)
```

Comment:
Related to our discussion from Monday, since now we may define the address and path of every endpoint, we can essentially get around the `/1.0` prefix restriction correct? Maybe that's a good enough approach for lxd site manager. Originally we were thinking of grouping some of the endpoints (management API e.g. `/oidc/*`, `/sites`) with the core API but maybe we can do with an additional API?

Reply:
Yep but as discussed this would mean having three listeners, one for the microcluster core API, one for the site management API, and one for the heartbeat API.

Reply:
No worries :)